src/patternfly/components/Chip/chip.scss

```diff
@@ -42,7 +42,7 @@
   display: inline-flex;
   align-items: center;
   padding-left: var(--pf-c-chip--PaddingLeft);
-  color: var(--pf-c-chip--Color);
+  color: var(--pf-c-chip__text--Color);
```

Comment:
Pretty sure you can just remove this since `.pf-c-chip__text` is a required element of chip. @christiemolloy can you confirm?

Reply:
Yes `color: var(--pf-c-chip__text--Color);` is already on the `.pf-c-chip__text` so you can remove this line